lib/exception_notifier/datadog_notifier.rb

```diff
@@ -1,3 +1,5 @@
+require 'active_support/core_ext/time'
```

Comment:
don't we also need other extensions for `options.reverse_merge(default_options)` ? (maybe we should just change it to `default_options.reverse_merge(options)`?)

and we will probably need something for `ActionDispatch::Request.new(options[:env])` (maybe a `defined?(ActionDispatch::Request)`)